.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
```

Comment:
I would recommend using a similar file like [**this**](https://raw.githubusercontent.com/TheAlgorithms/C-Plus-Plus/master/.github/ISSUE_TEMPLATE/bug_report.yml) one.